src/Common/src/Interop/Linux/Interop.Libraries.cs

```diff
@@ -11,5 +11,6 @@ private static partial class Libraries
         internal const string INODE64SUFFIX = "";
 
         internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+	internal const string LibCurl = "libcurl.so.4";             // Curl HTTP client library
```

Comment:
The last libcurl release with soname=3 was 7.15.5 from August 2006 and it's replaced by the newer version practically everywhere. Ubuntu symlinks libcurl.so.3 -> libcurl.so.4, other distros don't even ship libcurl.so.3 anymore :)